README.md

```diff
@@ -9,3 +9,4 @@ Web application based on PHP, MySQL, and Javascript/AJAX, visually designed with
 3) Logout (Signout)
 4) Remember me
 5) Users and Remember Me database creation
+6)
```

Comment:
Did you intend to include more changes here?

Reply:
Not for now. I am trying to explain Ognjan how GitHub works.

Reply:
Ah, ok. Welcome Ognjan! :)

Reply:
Thank you, Ralph. :)